src/main/java/org/fungover/mmotodo/entities/task/Task.java

```diff
@@ -0,0 +1,81 @@
+package org.fungover.mmotodo.entities.task;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import org.fungover.mmotodo.entities.category.Category;
+import org.fungover.mmotodo.entities.tag.Tag;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.time.Instant;
+import java.util.Objects;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "task")
+public class Task {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Integer id;
+
+    @Size(max = 255)
+    @NotNull
+    @Column(name = "title", nullable = false)
+    private String title;
+
+    @Size(max = 255)
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "created")
+    @CreationTimestamp
+    private Instant created;
+
+    @Column(name = "updated")
+    @UpdateTimestamp
+    private Instant updated;
+
+    @NotNull
+    @Column(name = "time_estimation", nullable = false)
+    private Double timeEstimation;
+
```

Comment:
This is a bit ambiguous, using a Double for timeEstimation. What would 10.5 mean, for instance?